fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -4158,20 +4157,12 @@ public LocalDate getExpectedDisbursedOnLocalDateForTemplate() {
         return expectedDisbursementDate;
     }
 
-    /*
-     * Reason for derving
-     */
-
     public BigDecimal getDisburseAmountForTemplate() {
-        BigDecimal principal = this.loanRepaymentScheduleDetail.getPrincipal().getAmount();
```

Comment:
How does this change relates to the "undo last disbursal function" fix?

Reply:
When you get the disburse template you get a wrong amount after the first disbursement

Reply:
I see. Thanks for letting me know.
Would you please raise a new story and PR with this change?

Reply:
Done